wai-handler-hal.cabal

```
@@ -50,13 +50,13 @@ common deps
   build-depends:
     , base                  >=4.12      && <4.19
     , base64-bytestring     >=1.0.0.0   && <1.3
-    , bytestring            >=0.10.8    && <0.12.1
+    , bytestring            >=0.10.8    && <0.12.2
     , case-insensitive      ^>=1.2.0.0
     , hal                   >=0.4.7     && <0.4.11 || >=1.0.0 && <1.2
     , http-media            ^>=0.8.1.1
     , http-types            ^>=0.12.3
     , network               >=2.8.0.0   && <3.2
-    , text                  ^>=1.2.3    || >=2.0   && <2.1.1
+    , text                  ^>=1.2.3    || >=2.0   && <2.1.2
```

```suggestion
    , text                  ^>=1.2.3    || >=2.0   && <2.2
```
For the same reason.